### DISCUSSION TITLE
multi-step API , custom conector

### DISCUSSION THREAD

**Discussion by torresftw** [airbytehq/airbyte#33771] [Connector Questions]
I'm working on making a public API work with Airbyte. To get the real data, I need to call the API four times. I've set up Airbyte locally using Docker, created a custom connector, and added the first stream. Now, how do I set up the rest of the streams so they continue from the previous one and get the actual data? Any help on achieving this would be appreciated!

**Comment by marcosmarxm** [comment]
What CDK are you using? low-code, connector builder or python cdk?

[a comment by an automated account is not shown]
